tests/Integrations/Predis/PredisTest.php

```diff
@@ -167,12 +168,18 @@ public function testPredisPipeline()
             $this->assertInstanceOf('Predis\Response\Status', $responseFlush);
         });
 
-        $this->assertSpans($traces, [
+        if (Versions::phpVersionMatches('5') && static::IS_SANDBOX) {
```

Comment:
The previous code seemed to work on PHP 5; why is PHP 5 + sandbox special cased?

Reply:
Because it was counting the number of items in the queue before forwarding the real call.
If you do the posthook (as the prehook is not supported in php 5 for the sandboxed api) by then is already empty.